posthog/models/plugin.py

```diff
@@ -340,3 +340,16 @@ def plugin_config_reload_needed(sender, instance, created=None, **kwargs):
 @receiver([post_save, post_delete], sender=PluginAttachment)
 def plugin_attachement_reload_needed(sender, instance, created=None, **kwargs):
     reload_plugins_on_workers()
+
+
+class PluginSourceFile(UUIDModel):
```

Comment:
Code style: Please order models first, receivers second. Current order makes it easy to miss the receiver just above.